LoopKitTests/TemporaryScheduleOverrideHistoryTests.swift

```diff
@@ -221,8 +221,8 @@ final class TemporaryScheduleOverrideHistoryTests: XCTestCase {
             RepeatingScheduleValue(startTime: .hours(0), value: 1.8),
             RepeatingScheduleValue(startTime: .hours(4), value: 1.2),
             RepeatingScheduleValue(startTime: .hours(6), value: 1.4),
-            RepeatingScheduleValue(startTime: .hours(20), value: 1.0),
-            RepeatingScheduleValue(startTime: .hours(22), value: 1.5),
+            RepeatingScheduleValue(startTime: .hours(20), value: 1.5),
+            //RepeatingScheduleValue(startTime: .hours(22), value: 1.5),
         ])!
 
         print(expected)
```

Comment:
(If you wouldn't mind, you can delete this old print statement too, I must've missed it when I wrote this originally!)